DemoApp-Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/data/home/AdType.kt

```diff
@@ -0,0 +1,13 @@
+package com.applovin.enterprise.apps.demoapp.data.home
+
+import android.content.Intent
+
+/**
+ * A [ListItem] representing an ad type on the home screen.
+ * <p>
+ * Created by Harry Arakkal on 09/17/2019.
+ */
+class AdType(val adType: String, val intent: Intent) : ListItem
```

Comment:
I think we should make this and `SectionHeader` a [`data class`](https://kotlinlang.org/docs/reference/data-classes.html)